templates/zerver/api/incoming-webhooks-walkthrough.md

```diff
@@ -570,7 +570,7 @@ using `REQ` without any special handling. How does this work in a test?
 
 The new attribute `TOPIC` exists only in our class so far. In order to
 construct a URL with a query parameter for `topic`, you can pass the
-attribute `TOPIC` as a keyword argument to `build_webhook_url`, like so:
+attribute `TOPIC` as a keyword parameter to `build_webhook_url`, like so:
```

Comment:
The usage of `argument` is actually correct here in the context of Python code, so it doesn't need to be replaced here.